client/components/layout/MainLayout.tsx

```diff
@@ -211,15 +213,12 @@ const MainLayout: React.FC = () => {
           <Box></Box>
         )}
 
-        <Box
-          className="router-outlet"
-          sx={{ display: 'flex' }}
-        >
+        <Box className="router-outlet">
           <Typography
```

Comment:
@lomeintheory I think you want to remove the Typography element here and style your text within the PromptInputForm component.